packages/redux/src/bags/actions/factories/fetchBagOperationFactory.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Bag,
+  BagOperation,
+  Config,
+  GetBagOperation,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import {
+  FETCH_BAG_OPERATION_FAILURE,
+  FETCH_BAG_OPERATION_REQUEST,
+  FETCH_BAG_OPERATION_SUCCESS,
+} from '../../actionTypes';
+import { normalize } from 'normalizr';
+import bagOperationSchema from '../../../entities/schemas/bagOperation';
+import type { Dispatch } from 'redux';
+
+/**
+ * Creates a thunk factory configured with the specified client to fetch a bag operation with given id.
+ *
+ *
+ * @param  getBagOperation - Get bag operation client.
+ *
+ * @returns  Thunk factory.
+ */
+const fetchBagOperationFactory =
+  (getBagOperation: GetBagOperation) =>
+  (bagId: Bag['id'], bagOperationId: BagOperation['id'], config?: Config) =>
+  async (dispatch: Dispatch): Promise<BagOperation> => {
+    try {
+      dispatch({
+        type: FETCH_BAG_OPERATION_REQUEST,
+        payload: { bagOperationId },
```

Comment:
In other actions, we use the `meta` property instead of the `payload` for passing this kind of information (You can check the `addBagItemFactory` for example.